src/Discord/Parts/Guild/WelcomeScreen.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is a part of the DiscordPHP project.
+ *
+ * Copyright (c) 2015-present David Cole <david.cole1340@gmail.com>
+ *
+ * This file is subject to the MIT license that is bundled
+ * with this source code in the LICENSE.md file.
+ */
+
+namespace Discord\Parts\Guild;
+
+use Discord\Parts\Part;
+
+/**
+ * A Welcome Screen of a Guild
+ *
+ * @link https://discord.com/developers/docs/resources/guild#welcome-screen-object-welcome-screen-structure
+ * 
+ * @property string                      $description      The server description shown in the welcome screen.
+ * @property Collection|WelcomeChannel[] $welcome_channels The channels shown in the welcome screen, up to 5.
```

Comment:
Need a `getWelcomeChannels()` function to return a collection and/or an array of parts